eng/versioning/external_dependencies.txt

```diff
@@ -171,11 +171,20 @@ applicationinsights_com.microsoft.azure:azure-client-runtime;1.6.3
 applicationinsights_com.microsoft.azure:azure-client-authentication;1.6.3
 
 # everything under sdk\cosmos
-cosmos_org.mockito:mockito-core;1.10.19
+cosmos_com.fasterxml.jackson.module:jackson-module-afterburner;2.10.1
 cosmos_com.google.guava:guava;25.0-jre
 cosmos_io.dropwizard.metrics:metrics-core;4.1.0
-cosmos_io.dropwizard.metrics:metrics-jvm;4.1.0
 cosmos_io.dropwizard.metrics:metrics-graphite;4.1.0
+cosmos_io.dropwizard.metrics:metrics-jvm;4.1.0
```

Comment:
/CC @joshfree @alzimmermsft 
Alan and I have chatted about these netty dependencies. Right now the Cosmos io.netty dependencies are ahead of the common versions. We should look and see if we can bring the common dependencies up to these versions.

Reply:
I'm good with upgrading to these newer versions if we get a few good test runs in with them.